01/RitamDey/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    words = []
+    with open(DICTIONARY) as dictionary:
+        words = [word.strip("\n") for word in dictionary]
+    return words
+
+def calc_word_value(word: str):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    total_sum = 0
+    for char in word:
+        if char.upper() in LETTER_SCORES:
+            total_sum += LETTER_SCORES[char.upper()]
```

Comment:
you can use a list comprehension like in line 7 to merge these four lines into one with something like `sum(LETTER_SCORES.get(char.upper(), 0) for char in word)`. Notice my use of `get` which is another very very Pythonic solution for not writing `if key in dict do...else` each time. It provides a default value if the key is not found.